Pipfile

```
@@ -34,7 +34,7 @@ werkzeug = "~=2.3"
 wtforms = "~=3.0"
 newrelic = "*"
 flask-talisman = "*"
-notifications-utils = {editable = true, ref = "main", git = "https://github.com/GSA/notifications-utils.git"}
+notifications-utils = {editable = true, ref = "notify-512", git = "https://github.com/GSA/notifications-utils.git"}
```

Once https://github.com/GSA/notifications-utils/pull/39 is merged, this will h ave to be changed back to `main`:

```suggestion
notifications-utils = {editable = true, ref = "main", git = "https://github.com/GSA/notifications-utils.git"}
```